.github/workflows/ci-testing.yml

```diff
@@ -63,7 +63,9 @@ jobs:
           pip list
 
       - name: Tests
-        run: coverage run --source litserve -m pytest src tests -v
+        run: |
+          python -m pytest --cov=litserve src/ tests/ -v -s
```

Comment:
```suggestion
          python -m pytest --cov=litserve src/ tests/ -v --capture=no
```

Reply:
@Borda found out the reason CI seem to be stuck - https://github.com/pytest-dev/pytest-asyncio/issues/179

Reply:
also could be related to https://github.com/miguelgrinberg/python-socketio/issues/263

Reply:
btw, I can reproduce the hanging also locally

Reply:
thanks for checking @Borda! yeah, I can also reproduce locally now. [This](https://github.com/Lightning-AI/LitServe/blob/f6833237215cc62d8dafc0d2280fb2306879d0f9/tests/test_lit_server.py#L402-L405) particular block of test is making CI fail.

Reply:
https://github.com/Lightning-AI/LitServe/blob/f6833237215cc62d8dafc0d2280fb2306879d0f9/tests/test_lit_server.py#L402-L405